config/config.go

```diff
@@ -35,6 +35,16 @@ func (c *Config) Get(name string) (*Command , error){
 	return nil, errors.New("")
 }
 
+func (c *Config) Position(name string) (int){
```

Comment:
Given now we have unique names for each command, is better if you manage all the stuff related to the config file inside the config.go file, so, a method like
```
func (c *Config) UpdateCommand(name string, cmd *Command) error {
    ....
}
```
will be easy to maintain